.circleci/config.yml

```diff
@@ -19,12 +19,6 @@ workflows:
 
     jobs:
       - setup_and_typescript_tests
```

Comment:
Do we still need this file? [I'm assuming we run typescript test in all the e2e test files, right?

Reply:
Yes, we still need it; I don't think typescript tests are run in e2e test files.